app.js

```
@@ -1,21 +1,27 @@
+const init_db = require('./model/init_db');
+const login = require('./routes/login');
+const products = require('./routes/products');
 var init_db = require('./model/init_db');
 var login = require('./routes/login');
 var products = require('./routes/products');
-
+log4js.addAppender(log4js.appenders.file('app-custom.log'), 'vnode');
 var app = express();
 
-// config second logger
+logger4js.setLevel('INFO');
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)